### DISCUSSION TITLE
[gatsby-plugin-image] layout: FLUID doesn't work

### DISCUSSION THREAD

**Discussion by adiusz** [gatsbyjs/gatsby#31349] [Help] [answered]
I copied this query from https://www.gatsbyjs.com/plugins/@gromy/gatsby-plugin-image/#graphql-resolver

```
coverImage: file(relativePath: { eq: "plant.jpg" }) {
  childImageSharp {
    gatsbyImageData(maxWidth: 720, layout: FLUID, placeholder: TRACED_SVG)
  }
}
```

and got this error:
```
Unknown argument "maxWidth" on field "ImageSharp.gatsbyImageData". Did you
mean "width"?  graphql/template-strings
```

When I replace `FLUID` with `FIXED` and `maxWidth` with `width` everything works fine, except the image is not displaying properly.

Note: I'm using background image example from here: https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/#background-images

**Comment by LekoArts** [comment, marked as answer]
You're referencing the wrong package (an outdated fork). Here's the correct package and its README: https://www.gatsbyjs.com/plugins/gatsby-plugin-image/

**Comment by adiusz** [comment]
I'm wondering why you don't put some note on that article's top that it's outdated. Would be a huge help. 
Anyway thanks for helping

**Reply by LekoArts** [reply]
It's nothing we can fix, the npm usee gromy needs to remove their forked published package
